std-bits/table/src/main/java/org/enso/table/operations/AddRunning.java

```diff
@@ -41,38 +43,92 @@ private static RunningStatistic<Double> createRunningStatistic(
         return new RunningMeanStatistic(sourceColumn, problemAggregator);
       }
       case Minimum -> {
+        if (sourceColumn.getStorage().getType() == IntegerType.INT_64) {
```

Comment:
What about all other integer types like `INT_16`, or `BigIntegerType`?

Reply:
Do we plan to handle such cases in a separate PR?

We should also consider other types, like `DateType` etc. - they can also be compared.

IIRC `Table.aggregate` with `Minimum` should work for Dates/Times (but I think it may be untested at present unfortunately). Ideally, the running aggregation should probably work with them as well.

Reply:
If we don't support these _for now_, I think it would be good to at least add a 'negative' test checking how the method fails if trying to run a running Minimum over a date column - it should ideally be some clear 'unsupported' error message.

Reply:
This also made me curious (a bit tangentially related to this PR) - what does the failure look like when e.g. a Text column is given to a `Mean` statistic?

Reply:
Added a test for trying to sum a text column.

Reply:
Actually there already was one:
```
        group_builder.specify "Running sum does not work over a string column" <|
            (table.running Statistic.Sum "Passenger").should_fail_with Invalid_Value_Type
```

Reply:
Added support for the IntegerType.* types

Reply:
I am going to do BigIntegerType and Datetime as future MRs.